### PR TITLE
issue #1186 - minor javadoc update

### PR DESCRIPTION
**Pull request by lmsurpre**
super minor

Signed-off-by: Lee Surprenant <lmsurpre@us.ibm.com>

### CODE REVIEW

File: fhir-model/src/main/java/com/ibm/fhir/model/type/Base64Binary.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ private Base64Binary(Builder builder) {
 
     /**
      * The actual value
-     * 
+     *
```
Comment (prb112) [inline fhir-model/src/main/java/com/ibm/fhir/model/type/Base64Binary.java:36]: I'm happy to approve this... however... This is a generated class, perhaps the generator updates are missing?

Comment (lmsurpre) [reply fhir-model/src/main/java/com/ibm/fhir/model/type/Base64Binary.java:36]: good call

**Review by prb112 — APPROVED** [review]

**Comment by JohnTimm** [comment]
With the `Base64Binary` class we have this relationship:

```
Base 64 encoded string (java.lang.String) <-----> Unencoded binary data (byte[])
```

My preference would be to using a naming convention that captures that relationship. So for the `of` factory method that sets the binary data directly we would say something like:

```
Factory method for creating Base64Binary objects from a byte array of unencoded binary data

@param
    a byte array containing unencoded binary data
```

**Comment by lmsurpre** [comment]
@JohnTimm you had previously approved https://github.com/IBM/FHIR/pull/2157 and so I assumed you were fine with what was there.  Based on that, I closed #1186.  Feel free to open a new issue and/or PR if you'd like to see further changes here.

**Pull request merged**